README.md

```diff
@@ -14,7 +14,13 @@ The static marketing site for login.gov
 
 ## Development
 
-This is a Jekyll-built static site. To start serve the site locally in development:
+This is a Jekyll-built static site. To install dependencies:
+
+```
+bundle install
```

Comment:
Looks like @zachmargolis added `setup` to the make file which runs
```
bundle check || bundle install
	npm install
```
would you mind updating this line to use that instead?